sdk/storage/storage-file-share/review/storage-file-share.api.md

```diff
@@ -98,6 +99,17 @@ export interface CommonOptions {
     tracingOptions?: OperationTracingOptions;
 }
 
+// @public
+export interface CopyFileSmbInfo {
```

Comment:
Consider rename to `FileSmbProperties` to be consistent with other languages.

Reply:
or add a wrapper type `FileSmbProperties` if it is easier.
Java has https://github.com/Azure/azure-sdk-for-java/pull/7589/files#diff-327dce8b4df9fed9844e4ff667950f82